package.json

```diff
@@ -37,6 +37,7 @@
     "server"
   ],
   "devDependencies": {
+    "file-loader": "^6.0.0",
```

Comment:
I believe that this should actually be in `client/package.json` and if you add it via `yarn add file-loader` from the `client` folder, I don't think it will generate a `package-lock.json`.